distributions/Jamulus.1

```diff
@@ -60,82 +60,82 @@ is also possible.
 .Pp
 One participant starts
 .Nm
-in server mode, ideally on a dedicated server (virtual) machine;
-all participants start the (graphical) client which transmits audio
-to the server, receiving back a mixed stream.
+in Server mode, ideally on a dedicated Server (virtual) machine;
+all participants start the (graphical) Client which transmits audio
```

Comment:
lowercase

Reply:
Wrong.  The point of the PR is to capitalise "Server", "Client" and other relevant terms where referring to a running instance of Jamulus.